tt-rss: 2021-06-21 -> 2022-08-01 (#184496)
NixOS/nixpkgs: nixos/doc/manual/release-notes/rl-2211.section.md

- Add udev rules for the Teensy family of microcontrollers.

> - The `tt-rss` service performs two database migrations when you first use its web UI after upgrade; consider backing up its database before starting.

Reviewer: ```suggestion
- The `tt-rss` service will perform database migrations when first visiting the web UI after this upgrade. Consider backing up its database before updating.
```

Author: Sigh, time to polish my bikeshed paintbrush! I can see why you prefer "updating" to "starting" even though "starting" is more accurate (since it won't perform the migrations until you press Go in the UI). But "when first visiting" is strictly less clear than "when you first use" (or "visit"), because 1) the agent of "visiting" is not specified and is weakly implied by the grammar to be "the `tt-rss` service", which is of course nonsense, and 2) "visiting" is in a continuous tense, whereas the upgrade is performed once when you take a specific action.

Reviewer: > since it won't perform the migrations until you press Go in the UI

That was not obvious to me from your changelog entry. It sounded more like you must do it before upgrading your system or it is automatically done at startup or immediately when you open the first page in the browser.
